addons/account/wizard/setup_wizards.py

```diff
@@ -75,27 +75,16 @@ class SetupBarBankConfigWizard(models.TransientModel):
         comodel_name='account.journal', inverse='set_linked_journal_id',
         compute="_compute_linked_journal_id", check_company=True,
         domain="[('type','=','bank'), ('bank_account_id', '=', False), ('company_id', '=', company_id)]")
-    new_journal_code = fields.Char(string="Code", required=True, default=lambda self: self._onchange_new_journal_code())
```

Comment:
can't say I'm too certain of that change. if you have 2 bank accounts, you might want to define their codes with "CPH" and "BNP" instead of meaningless default "BNK1", "BNK2". So it seems important to me. Don't you think @tsb-odoo ?

Reply:
@qdp-odoo I do think it is important for the user to recognize his bank accounts by the name but not by the shortcode. Though, if really, the case you mention occurs. The user can still edit the short code in the journal configuration tab.

Reply:
the shortcode of the journal will also be used for the default naming of accounting entries created in that journal... BNK1/2020/01/0001 or CPH/2020/01/0001, I really think the latest makes more sense

Reply:
@qdp-odoo I agree the user might appreciate to see a meaningful shortcode. Though I don't think at the time of creating his first account, he has enough understanding of what he is doing for such a question. What if he has multiple account at CPH and so on??? IMHO, don't make him think at that moment. He can still resequence later the entries created in the journal when he figures out that the shortcode affects the naming of accounting entries.

Reply:
@vin-odoo To decide with @tsb-odoo

Reply:
@vin-odoo In order to remove frictions during the onboarding, let's remove the short code. And the user can still resequence later on when he understands the eddects of such a short code on the sequence numbers.